### PR TITLE
Adapt Serverless Tier in Developer Guide

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11862
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC-tidb-cloud.md [hunk 1]
```diff
@@ -17,11 +17,10 @@
...
     - [Build a TiDB Cluster in TiDB Cloud (Serverless Tier)](/develop/dev-guide-build-cluster-in-cloud.md)
     - [CRUD SQL in TiDB](/develop/dev-guide-tidb-crud-sql.md)
-    - Build a Simple CRUD App with TiDB
-      - [Java](/develop/dev-guide-sample-application-java.md)
-      - [Golang](/develop/dev-guide-sample-application-golang.md)
   - Example Applications
-    - [Build a TiDB Application using Spring Boot](/develop/dev-guide-sample-application-spring-boot.md)
+    - [Java](/develop/dev-guide-sample-application-java.md)
+    - [Golang](/develop/dev-guide-sample-application-golang.md)
+    - [Java (Spring Boot)](/develop/dev-guide-sample-application-spring-boot.md)
```
Comment (Oreoxmt) [inline TOC-tidb-cloud.md:23]:
```suggestion
    - [Java](/develop/dev-guide-sample-application-java.md)
    - [Java (Spring Boot)](/develop/dev-guide-sample-application-spring-boot.md)
    - [Golang](/develop/dev-guide-sample-application-golang.md)
```

File: TOC.md [hunk 2]
```diff
@@ -21,11 +21,10 @@
...
     - [Build a TiDB Cluster in TiDB Cloud (Serverless Tier)](/develop/dev-guide-build-cluster-in-cloud.md)
     - [CRUD SQL in TiDB](/develop/dev-guide-tidb-crud-sql.md)
-    - Build a Simple CRUD App with TiDB
-      - [Java](/develop/dev-guide-sample-application-java.md)
-      - [Golang](/develop/dev-guide-sample-application-golang.md)
   - Example Applications
-    - [Build a TiDB Application using Spring Boot](/develop/dev-guide-sample-application-spring-boot.md)
+    - [Java](/develop/dev-guide-sample-application-java.md)
+    - [Golang](/develop/dev-guide-sample-application-golang.md)
+    - [Java (Spring Boot)](/develop/dev-guide-sample-application-spring-boot.md)
```
Comment (Oreoxmt) [inline TOC.md:27]:
```suggestion
    - [Java (Spring Boot)](/develop/dev-guide-sample-application-spring-boot.md)
    - [Golang](/develop/dev-guide-sample-application-golang.md)
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 3]
```diff
@@ -146,55 +146,32 @@ mysql  Ver 15.1 Distrib 5.5.68-MariaDB, for Linux (x86_64) using readline 5.1
...
-
-  {{< copyable "shell-regular" >}}
-
-  ```shell
-  git clone https://github.com/pingcap-inc/tidb-example-java.git
-  ```
+> **Note:**
+>
+> - Use TLS connections here is [required](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters).
+> - If you are having issues with connecting, you can read TiDB Cloud [Serverless Tier Clusters](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters) for more information.
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:155]:
```suggestion
<CustomContent platform="tidb">

> **Note:**
>
> - The preceding example uses the TLS connection, which is [required](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters) when connecting a Serverless Tier cluster.
> - If you encounter problems when connecting to a Serverless Tier cluster, you can read [Secure Connections to Serverless Tier Clusters](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters) for more information.

</CustomContent>

<CustomContent platform="tidb-cloud">

> **Note:**
>
> - The preceding example uses the TLS connection, which is [required](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md) when connecting a Serverless Tier cluster.
> - If you encounter problems when connecting to a Serverless Tier cluster, you can read [Secure Connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md) for more information.

</CustomContent>
```

File: develop/dev-guide-sample-application-spring-boot.md [hunk 4]
```diff
@@ -206,13 +206,11 @@ spring:
       ddl-auto: create-drop
 ```
 
-If you set the password to `123456`, the connection string you get in TiDB Cloud is as follows:
+Suppose that the password you set is `123456`, and the connection parameters you get from TiDB Cloud Serverless Tier are the following:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-spring-boot.md:209]:
```suggestion
Suppose that the password you set is `123456`, and the connection parameters you get from the cluster details page are the following:
```

File: develop/dev-guide-sample-application-spring-boot.md [hunk 5]
```diff
@@ -188,7 +188,7 @@ If you want to learn more about the code of this application, refer to [Implemen
 
 ### Step 5.1 Change parameters
 
-If you use a non-local default cluster, a TiDB Cloud cluster or a remote cluster, change the `spring.datasource.url`, `spring.datasource.username`, `spring.datasource.password` parameters in the `application.yml` (located in `src/main/resources`).
+If you are using the TiDB Cloud Serverless Tier clusters, change the `spring.datasource.url`, `spring.datasource.username`, `spring.datasource.password` parameters in the `application.yml` (located in `src/main/resources`).
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-spring-boot.md:191]:
```suggestion
If you are using a TiDB Cloud Serverless Tier cluster, change the `spring.datasource.url`, `spring.datasource.username`, `spring.datasource.password` parameters in the `application.yml` (located in `src/main/resources`).
```

File: develop/dev-guide-sample-application-java.md [hunk 6]
```diff
@@ -1654,11 +1650,11 @@ mysqlDataSource.setUser("root");
 mysqlDataSource.setPassword("");
 ```
 
-Suppose that the password you set is `123456` and the connection string you get from TiDB Cloud is the following:
+Suppose that the password you set is `123456`, and the connection parameters you get from TiDB Cloud Serverless Tier are the following:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-java.md:1653]:
```suggestion
Suppose that the password you set is `123456`, and the connection parameters you get from the cluster details page are the following:
```

File: develop/dev-guide-sample-application-java.md [hunk 7]
```diff
@@ -1600,13 +1598,11 @@ If you are using a non-local default cluster, such as TiDB Cloud or other remote
 </hibernate-configuration>
 ```
 
-Suppose that the password you set is `123456` and the connection string you get from TiDB Cloud is the following:
-
-{{< copyable "shell-regular" >}}
+Suppose that the password you set is `123456`, and the connection parameters you get from TiDB Cloud Serverless Tier are the following:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-java.md:1601]:
```suggestion
Suppose that the password you set is `123456`, and the connection parameters you get from the cluster details page are the following:
```

File: develop/dev-guide-sample-application-java.md [hunk 8]
```diff
@@ -1534,13 +1534,11 @@ If you are using a non-local default cluster, such as TiDB Cloud or other remote
 </configuration>
 ```
 
-Suppose that the password you set is `123456` and the connection string you get from TiDB Cloud is the following:
+Suppose that the password you set is `123456`, and the connection parameters you get from TiDB Cloud Serverless Tier are the following:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-java.md:1537]:
```suggestion
Suppose that the password you set is `123456`, and the connection parameters you get from the cluster details page are the following:
```

File: develop/dev-guide-sample-application-golang.md [hunk 9]
```diff
@@ -793,52 +793,60 @@ When using go-sql-driver/mysql, you need to connect to your cluster and run the
...
-
-{{< copyable "" >}}
+If you are using the TiDB Cloud Serverless Tier clusters, modify the value of the `dsn` in `sqldriver.go`:
 
 ```go
 dsn := "root:@tcp(127.0.0.1:4000)/test?charset=utf8mb4"
 ```
 
-Suppose that the password you set is `123456` and the connection string you get from TiDB Cloud is the following:
+Suppose that the password you set is `123456`, and the connection parameters you get from TiDB Cloud Serverless Tier are the following:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-golang.md:833]:
```suggestion
Suppose that the password you set is `123456`, and the connection parameters you get from the cluster details page are the following:
```

File: develop/dev-guide-sample-application-golang.md [hunk 10]
```diff
@@ -793,52 +793,60 @@ When using go-sql-driver/mysql, you need to connect to your cluster and run the
...
 ```
 
 </div>
 
 <div label="Using go-sql-driver/mysql" value="sqldriver">
 
-If you are using a non-local default cluster, such as TiDB Cloud or other remote clusters, modify the value of the `dsn` in `sqldriver.go`:
-
-{{< copyable "" >}}
+If you are using the TiDB Cloud Serverless Tier clusters, modify the value of the `dsn` in `sqldriver.go`:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-golang.md:827]:
```suggestion
If you are using a TiDB Cloud Serverless Tier cluster, modify the value of the `dsn` in `sqldriver.go`:
```

File: develop/dev-guide-sample-application-golang.md [hunk 11]
```diff
@@ -793,52 +793,60 @@ When using go-sql-driver/mysql, you need to connect to your cluster and run the
...
+If you are using the TiDB Cloud Serverless Tier clusters, modify the value of the `dsn` in `gorm.go`:
 
 {{< copyable "" >}}
 
 ```go
 dsn := "root:@tcp(127.0.0.1:4000)/test?charset=utf8mb4"
 ```
 
-Suppose that the password you set is `123456` and the connection string you get from TiDB Cloud is the following:
+Suppose that the password you set is `123456`, and the connection parameters you get from TiDB Cloud Serverless Tier are the following:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-golang.md:804]:
```suggestion
Suppose that the password you set is `123456`, and the connection parameters you get from the cluster details page are the following:
```

File: develop/dev-guide-sample-application-golang.md [hunk 12]
```diff
@@ -793,52 +793,60 @@ When using go-sql-driver/mysql, you need to connect to your cluster and run the
 
 <div label="Using GORM (Recommended)" value="gorm">
 
-If you are using a non-local default cluster, such as TiDB Cloud or other remote clusters, modify the value of the `dsn` in `gorm.go`:
+If you are using the TiDB Cloud Serverless Tier clusters, modify the value of the `dsn` in `gorm.go`:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-golang.md:796]:
```suggestion
If you are using a TiDB Cloud Serverless Tier cluster, modify the value of the `dsn` in `gorm.go`:
```

File: develop/dev-guide-sample-application-java.md [hunk 13]
```diff
@@ -1489,7 +1489,7 @@ When using JDBC, you need to connect to your cluster and run the statement in th
 
 <div label="Using Mybatis (Recommended)" value="mybatis">
 
-If you are using a non-local default cluster, such as TiDB Cloud or other remote cluster, modify the `dataSource.url`, `dataSource.username`, `dataSource.password` in `mybatis-config.xml`.
+If you are using the TiDB Cloud Serverless Tier clusters, modify the `dataSource.url`, `dataSource.username`, `dataSource.password` in `mybatis-config.xml`.
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-java.md:1492]:
```suggestion
If you are using a TiDB Cloud Serverless Tier cluster, modify the `dataSource.url`, `dataSource.username`, `dataSource.password` in `mybatis-config.xml`.
```

File: develop/dev-guide-sample-application-java.md [hunk 14]
```diff
@@ -1570,7 +1568,7 @@ In this case, you can modify the parameters in `dataSource` node as follows:
 
 <div label="Using Hibernate (Recommended)" value="hibernate">
 
-If you are using a non-local default cluster, such as TiDB Cloud or other remote cluster, modify the `hibernate.connection.url`, `hibernate.connection.username`, `hibernate.connection.password` in `hibernate.cfg.xml`.
+If you are using the TiDB Cloud Serverless Tier clusters, modify the `hibernate.connection.url`, `hibernate.connection.username`, `hibernate.connection.password` in `hibernate.cfg.xml`.
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-java.md:1571]:
```suggestion
If you are using a TiDB Cloud Serverless Tier cluster, modify the `hibernate.connection.url`, `hibernate.connection.username`, `hibernate.connection.password` in `hibernate.cfg.xml`.
```

File: develop/dev-guide-sample-application-java.md [hunk 15]
```diff
@@ -1642,7 +1638,7 @@ In this case, you can modify the parameters as follows:
 
 <div label="Using JDBC" value="jdbc">
 
-If you are using a non-local default cluster, such as TiDB Cloud or other remote clusters, modify the parameters of the host, port, user, and password in `JDBCExample.java`:
+If you are using the TiDB Cloud Serverless Tier clusters, modify the parameters of the host, port, user, and password in `JDBCExample.java`:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-java.md:1641]:
```suggestion
If you are using a TiDB Cloud Serverless Tier cluster, modify the parameters of the host, port, user, and password in `JDBCExample.java`:
```

File: develop/dev-guide-sample-application-golang.md [hunk 16]
```diff
@@ -793,52 +793,60 @@ When using go-sql-driver/mysql, you need to connect to your cluster and run the
...
 
-```
-mysql --connect-timeout 15 -u root -h xxx.tidbcloud.com -P 4000 -p
-```
+- Endpoint: `xxx.tidbcloud.com`
+- Port: `4000`
+- User: `2aEp24QWEDLqRFs.root`
 
-In this case, you can modify the parameters as follows:
+In this case, you can modify the code block as follows:
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-golang.md:810]:
```suggestion
In this case, you can modify the `mysql.RegisterTLSConfig` and `dsn` as follows:
```

File: develop/dev-guide-sample-application-golang.md [hunk 17]
```diff
@@ -793,52 +793,60 @@ When using go-sql-driver/mysql, you need to connect to your cluster and run the
...
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-golang.md:839]:
```suggestion
In this case, you can modify the `mysql.RegisterTLSConfig` and `dsn` as follows:
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 18]
```diff
@@ -146,55 +146,32 @@ mysql  Ver 15.1 Distrib 5.5.68-MariaDB, for Linux (x86_64) using readline 5.1
...
-
-    Suppose that the password you set is `123456` and the connection string you get from TiDB Cloud is the following:
-
-    {{< copyable "" >}}
+3. Fill in the password to sign in.
 
-    ```shell
-    mysql --connect-timeout 15 -u '4JC1i9KroBMFRwW.root' -h xxx.tidbcloud.com -P 4000 -D test -p
-    ```
+## Step 3. Run a SQL
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:159]:
```suggestion
## Step 3. Execute a SQL statement
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 19]
```diff
@@ -146,55 +146,32 @@ mysql  Ver 15.1 Distrib 5.5.68-MariaDB, for Linux (x86_64) using readline 5.1
...
 
-    In this case, you can modify the parameters as follows:
+Let's try to run your first SQL on TiDB Cloud.
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:161]:
```suggestion
Let's try to execute your first SQL statement on TiDB Cloud.
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 20]
```diff
@@ -146,55 +146,32 @@ mysql  Ver 15.1 Distrib 5.5.68-MariaDB, for Linux (x86_64) using readline 5.1
...
+```
 
-    ```java
-    mysqlDataSource.setServerName("xxx.tidbcloud.com");
-    mysqlDataSource.setPortNumber(4000);
-    mysqlDataSource.setDatabaseName("test");
-    mysqlDataSource.setUser("4JC1i9KroBMFRwW.root");
-    mysqlDataSource.setPassword("123456");
-    ```
+You will see output like this.
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:167]:
```suggestion
Expected output:
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 21]
```diff
@@ -146,55 +146,32 @@ mysql  Ver 15.1 Distrib 5.5.68-MariaDB, for Linux (x86_64) using readline 5.1
...
+```sql
++-------------------+
+| Hello TiDB Cloud! |
++-------------------+
+| Hello TiDB Cloud! |
++-------------------+
+```
 
-  Here is an example of the [expected output](https://github.com/pingcap-inc/tidb-example-java/blob/main/Expected-Output.md#plain-java-jdbc).
+If you see output like this. **_CONGRATULATIONS_**! You have successfully run your own SQL on TiDB Cloud.
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:177]:
```suggestion
If your actual output is similar to the expected output, congratulations, you have successfully execute a SQL statement on TiDB Cloud.
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 22]
```diff
@@ -146,55 +146,45 @@ mysql  Ver 15.1 Distrib 5.5.68-MariaDB, for Linux (x86_64) using readline 5.1
...
-
-  ```shell
-  git clone https://github.com/pingcap-inc/tidb-example-java.git
-  ```
+<CustomContent platform="tidb">
 
-2. Change connection parameters.
+> **Note:**
+>
+> - The preceding example uses the TLS connection, which is [required](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters) when connecting a Serverless Tier cluster.
```
Comment (ran-huang) [inline develop/dev-guide-build-cluster-in-cloud.md:156]:
```suggestion
> - When you connect to a Serverless Tier cluster, you must [use the TLS connection](https://docs.pingcap.com/tidbcloud/secure-connections-to-serverless-tier-clusters). 
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 23]
```diff
@@ -146,55 +146,45 @@ mysql  Ver 15.1 Distrib 5.5.68-MariaDB, for Linux (x86_64) using readline 5.1
...
-    ```java
-    mysqlDataSource.setServerName("localhost");
-    mysqlDataSource.setPortNumber(4000);
-    mysqlDataSource.setDatabaseName("test");
-    mysqlDataSource.setUser("root");
-    mysqlDataSource.setPassword("");
-    ```
+> **Note:**
+>
+> - The preceding example uses the TLS connection, which is [required](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md) when connecting a Serverless Tier cluster.
```
Comment (ran-huang) [inline develop/dev-guide-build-cluster-in-cloud.md:165]:
```suggestion
> - When you connect to a Serverless Tier cluster, you must [use the TLS connection](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).
```

File: develop/dev-guide-sample-application-java.md [hunk 24]
```diff
@@ -1586,7 +1584,7 @@ If you are using a non-local default cluster, such as TiDB Cloud or other remote
         <property name="hibernate.connection.driver_class">com.mysql.cj.jdbc.Driver</property>
         <property name="hibernate.dialect">org.hibernate.dialect.TiDBDialect</property>
         <property name="hibernate.connection.url">jdbc:mysql://localhost:4000/test</property>
-        <property name="hibernate.connection.username">root</property>
+        <property name="hibernate.connection.username">2aEp24QWEDLqRFs.root</property>
```
Comment (ran-huang) [inline develop/dev-guide-sample-application-java.md:1587]: 中文的 [L1652](https://github.com/pingcap/docs-cn/pull/11862/files#diff-2a24636c7b5fdfd7fac81070334b84a682582854b1b2ae1b6ba77249deb328fdR1652) 没有改 username

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 25]
```diff
@@ -23,24 +23,19 @@ This document walks you through the quickest way to get started with TiDB Cloud.
...
-6. After creating a cluster, click **Connect** in the upper-right corner. A connection dialog box is displayed.
+    Your TiDB Cloud cluster will be created in approximately 30 seconds.
 
-    > **Tip:**
-    >
-    > Alternatively, you can also click the name of your newly created cluster to go to the cluster details page, and then click **Connect** in the upper-right corner.
+6. After your TiDB Cloud cluster is created, click **Security Settings**. In the **Security Settings** dialog box, set a root password to connect to your cluster, and then click **Apply**. If you do not set a root password, you cannot connect to the cluster.
 
-7. In the dialog box, locate **Step 2: Connect with a SQL client**, and then copy the string to connect with a SQL client for later use.
+7. Then click **Connect**. A connection dialog box is displayed. Under **Connect with a SQL Client** in the dialog, click the tab of your preferred connection method, and then save the corresponding connection string. The following section uses MySQL client as an example.
```
Comment (ran-huang) [inline develop/dev-guide-build-cluster-in-cloud.md:38]:
```suggestion
7. Click **Connect**. A connection dialog box is displayed. Under **Connect with a SQL Client** in the dialog, click the tab of your preferred connection method, and then save the corresponding connection string. The following section uses MySQL client as an example.
```

File: develop/dev-guide-sample-application-java.md [hunk 26]
```diff
@@ -1557,8 +1555,8 @@ In this case, you can modify the parameters in `dataSource` node as follows:
             <!-- Database pool -->
             <dataSource type="POOLED">
                 <property name="driver" value="com.mysql.jdbc.Driver"/>
-                <property name="url" value="jdbc:mysql://xxx.tidbcloud.com:4000/test"/>
-                <property name="username" value="root"/>
+                <property name="url" value="jdbc:mysql://xxx.tidbcloud.com:4000/test?sslMode=VERIFY_IDENTITY&amp;enabledTLSProtocols=TLSv1.2,TLSv1.3"/>
+                <property name="username" value="2aEp24QWEDLqRFs.root"/>
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-java.md:1559]:
```suggestion
                <property name="username" value="root"/>
```

File: develop/dev-guide-sample-application-java.md (hunk 24 above)
Comment (Icemap) [reply develop/dev-guide-sample-application-java.md:1587]: 此处将改回 `root`，英文误修改了描述。

其描述行应为，“若使用 Serverless Tier，应从 `root` 改为 `<prefix>.root`”。
但此处写为：“若使用 Serverless Tier，应从 `<prefix>.root` 改为 `<prefix>.root`”。

Comment (Icemap) [inline develop/dev-guide-sample-application-java.md:1587]:
```suggestion
        <property name="hibernate.connection.username">root</property>
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 27]
```diff
@@ -23,24 +23,19 @@ This document walks you through the quickest way to get started with TiDB Cloud.
...
-    Your TiDB Cloud cluster will be created in approximately 5 to 15 minutes.
+4. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, and then select the region where you want to create your cluster.
 
-6. After creating a cluster, click **Connect** in the upper-right corner. A connection dialog box is displayed.
+    Your TiDB Cloud cluster will be created in approximately 30 seconds.
 
-    > **Tip:**
-    >
-    > Alternatively, you can also click the name of your newly created cluster to go to the cluster details page, and then click **Connect** in the upper-right corner.
+6. After your TiDB Cloud cluster is created, click **Security Settings**. In the **Security Settings** dialog box, set a root password to connect to your cluster, and then click **Apply**. If you do not set a root password, you cannot connect to the cluster.
```
Comment (ran-huang) [inline develop/dev-guide-build-cluster-in-cloud.md:36]:
```suggestion
6. After your TiDB Cloud cluster is created, click **Security Settings**. In the **Security Settings** dialog box, set a root password to connect to your cluster, and then click **Submit**. If you do not set a root password, you cannot connect to the cluster.
```

File: develop/dev-guide-sample-application-java.md [hunk 28]
```diff
@@ -1557,7 +1555,7 @@ In this case, you can modify the parameters in `dataSource` node as follows:
             <!-- Database pool -->
             <dataSource type="POOLED">
                 <property name="driver" value="com.mysql.jdbc.Driver"/>
-                <property name="url" value="jdbc:mysql://xxx.tidbcloud.com:4000/test"/>
+                <property name="url" value="jdbc:mysql://xxx.tidbcloud.com:4000/test?sslMode=VERIFY_IDENTITY&amp;enabledTLSProtocols=TLSv1.2,TLSv1.3"/>
                 <property name="username" value="root"/>
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-java.md:1559]:
```suggestion
                <property name="username" value="2aEp24QWEDLqRFs.root"/>
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**